dmapiclient/search.py

```diff
@@ -23,10 +48,22 @@ def get_url(self, path, index, q, page=None, aggregations=[], **filters):
         elif page:
             params['page'] = page
 
-        self._add_filters_to_params(params, filters)
+        self._add_filters_prefix_to_params(params, filters)
 
         return self._build_url(url=self._url(index=index, path=path), params=params)
 
+    def deconstruct_url(self, search_api_url):
```

Comment:
Just looking at this again, we could probably name this slightly better.

Reply:
We could!